internal/aws/awsutil/conn.go

```diff
@@ -301,13 +308,13 @@ func (c *Conn) newAWSSession(logger *zap.Logger, roleArn string, region string,
 // getSTSCreds gets STS credentials from regional endpoint. ErrCodeRegionDisabledException is received if the
 // STS regional endpoint is disabled. In this case STS credentials are fetched from STS primary regional endpoint
 // in the respective AWS partition.
-func getSTSCreds(logger *zap.Logger, region string, roleArn string) (*credentials.Credentials, error) {
-	t, err := GetDefaultSession(logger, session.Options{})
+func getSTSCreds(logger *zap.Logger, region string, cfg *AWSSessionSettings) (*credentials.Credentials, error) {
```

Comment:
Can we revert the changes to `getSTSCreds` since we're no longer using it?

Reply:
I made a change to GetDefaultSession